scalene/scalene_preload.py

```diff
@@ -43,15 +63,12 @@ def setup_preload(args: argparse.Namespace) -> bool:
         except:
             pass
 
-        # Load the shared object on Linux.
-        if sys.platform == "linux":
-            if ("LD_PRELOAD" not in os.environ) and (
-                "PYTHONMALLOC" not in os.environ
-            ):
-                os.environ["LD_PRELOAD"] = os.path.join(
-                    os.path.dirname(scalene.__path__[0]), "libscalene.so"
-                )
-                os.environ["PYTHONMALLOC"] = "malloc"
+        # Start a subprocess with the required environment variables.
+        if sys.platform == "linux" or sys.platform == "darwin":
```

Comment:
AFAICT this test is redundant (already in the `get_preload_environ` call).

Reply:
Right, I'll remove. I thought of it, but was thinking we'll need to add something different for Windows support.